tutorials/migration_actor_api.md

```diff
@@ -0,0 +1,247 @@
+\page migrationactorapi
+
+# Migration from Gazebo-classic: Actor API
+
+When migrating plugins from Gazebo-classic to Gazebo, developers will
+notice that the C++ APIs for both simulators are quite different. Be sure to
+check the [plugin migration tutorial](migrationplugins.html) to get a high-level
+view of the architecture differences before using this guide.
+
+This tutorial is meant to serve as a reference guide for developers migrating
+functions from the
+[gazebo::phyiscs::Actor](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Actor.html)
+class.
+
+If you're trying to use some API which doesn't have an equivalent on Gazebo
+yet, feel free to
+[ticket an issue](https://github.com/gazebosim/gz-sim/issues/).
+
+## Actor API
+
+Gazebo-classic's `gazebo::physics::Actor` provides lots of functionality, which
+can be divided in these categories:
+
+* **Properties**: Setting / getting properties
+    * Example: [Actor::GetName](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Base.html#a9a98946a64f3893b085f650932c9dfee) / [Actor::SetName](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Entity.html#a5d74ac4d7a230aed1ab4b11933b16e92)
+* **Read family**: Getting children and parent
+    * Example: [Actor::GetParentModel](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Entity.html#a94d6f01102b5d949006fba3628d9f355)
+* **Write family**: Adding children, changing parent
+    * Example: [Actor::RemoveChildren](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Base.html#aa85d2386e6fb02bdbec060a74b63238a)
+* **Lifecycle**: Functions to control the actor's lifecycle
+    * Example: [Actor::Init](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Actor.html#ae048ef824aaf614707c1496a2aefd415)
+* **Others**: Functions that don't fit any of the categories above
+    * Example: [Actor::PlaceOnEntity](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Entity.html#a9ecbfeb56940cacd75f55bed6aa9fcb4)
+
+You'll find the Gazebo APIs below on the following headers:
+
+* [ignition/gazebo/Actor.hh](https://gazebosim.org/api/gazebo/6/Actor_8hh.html)
```

Comment:
There are a lot of instances of `ignition` and the URLs all point to 6.x versions.  I believe that they should be addressed as part of the forward port here.

Reply:
Done in [e42d913](https://github.com/gazebosim/gz-sim/pull/1977/commits/e42d9132b76be4d5c7a89d972223c0aef15e6a26)